README.md

```diff
@@ -1,2 +1,9 @@
 # simple-chat
 A simple chat app using Angular and socket.io
+
+# License
```

Comment:
Please make it a h2 tag by adding another #

Reply:
Done.